File: Quiz_ObjC/MainViewController.m
#import "MainViewController.h"

@interface MainViewController ()
@property(nonatomic,weak) IBOutlet UILabel* questionLabel;
@property(nonatomic,weak) IBOutlet UILabel* scoreLabel;
@property(nonatomic,weak) IBOutlet UIView* progressView;
@property(nonatomic,weak) IBOutlet UILabel* progressLabel;
@end

@implementation MainViewController

-(IBAction)buttonPressed:(UIButton*)sender
{
    NSLog(@"ObjC Button Pressed: %@",[sender currentTitle]);
}

- (void)viewDidLoad {
    [super viewDidLoad];
}

@end

File: Quiz_Swift/MainViewController.swift
import UIKit

@objc(MainViewController)
class MainViewController: UIViewController {
    
    @IBOutlet
    weak var questionLabel: UILabel!
    
    @IBOutlet
    weak var scoreLabel: UILabel!
    
    @IBOutlet
    weak var progressView: UIView!
    
    @IBOutlet
    weak var progressLabel: UILabel!
    
    @IBAction
    func buttonPressed(_ sender: UIButton)  {
        print("Swift Button Pressed: \(sender.currentTitle ?? "")")
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
}

